k8s/charts/orders-migrations/.helmignore

```diff
@@ -0,0 +1,23 @@
+# Patterns to ignore when building packages.
```

Comment:
how important are these files? can't tell if they are just fluff from helmify or if it makes sense to have them throughout the code base?